lib/iris/analysis/cartography.py

```diff
@@ -70,14 +70,37 @@ def wrap_lons(lons, base, period):
 
 def unrotate_pole(rotated_lons, rotated_lats, pole_lon, pole_lat):
     """
-    Convert rotated-pole lons and lats to unrotated ones.
+    Convert arrays of rotated-pole longitudes and latitudes to unrotated
+    arrays of longitudes and latitudes. The values of ``pole_lon`` and
+    ``pole_lat`` should describe the location of the rotated pole that
+    describes the arrays of rotated-pole longitudes and latitudes.
+
+    As the arrays of rotated-pole longitudes and latitudes must describe a
+    rectilinear grid, the arrays of rotated-pole longitudes and latitudes must
+    be of the same length as each other.
 
     Example::
 
         lons, lats = unrotate_pole(grid_lons, grid_lats, pole_lon, pole_lat)
```

Comment:
Good idea... how does `grid` --> `rotated` sound?

That is, `lons, lats = unrotate_pole(grid_lons, grid_lats, pole_lon, pole_lat)` would become `lons, lats = unrotate_pole(rotated_lons, rotated_lats, pole_lon, pole_lat)`, and so on.

This has the added benefit of matching to the signature for `unrotate_pole`, too.